FreeSurfer/LongitudinalFreeSurferPipeline.sh

```diff
@@ -48,111 +48,29 @@ source "${HCPPIPEDIR}/global/scripts/debug.shlib" "$@"           # Debugging fun
 opts_SetScriptDescription "Runs the Longitudinal FreeSurfer HCP pipeline"
 
 # Show usage information
 opts_AddMandatory '--subject' 'SubjectID' 'subject' "Subject ID (required)  Used with --path input to create full path to root directory for all outputs generated as path/subject"
-
 opts_AddMandatory '--path' 'StudyFolder' 'path' "Path to subject's data folder (required)  Used with --subject input to create full path to root directory for all outputs generated as path/subject)"
-
 opts_AddMandatory '--sessions' 'Sessions' 'sessions' "An @ symbol separated list of session IDs (required)"
-
 opts_AddMandatory '--template-id' 'TemplateID' 'template-id' "An @ symbol separated list of session IDs (required)"
-
+opts_AddMandatory '--use-T2w' 'UseT2w' 'UseT2w' "Set to 0 for no T2-weighted processing [1]" '1'
 opts_AddOptional '--seed' 'recon_all_seed' "Seed" 'recon-all seed value'
 
 #TSC: repeatable options aren't currently supported in newopts, do them manually and fake the help info for now
-opts_AddOptional '--extra-reconall-arg-base' 'extra_reconall_args_base' 'token' "(repeatable)  Generic single token argument to pass to recon-all for base template preparation.  Provides a mechanism to:  (i) customize the recon-all command  (ii) specify the recon-all stage(s) to be run (e.g., in the case of FreeSurfer edits)  If you want to avoid running all the stages inherent to the '-all' flag in recon-all,  you also need to include the --existing-subject flag.  The token itself may include dashes and equal signs (although Freesurfer doesn't currently use  equal signs in its argument specification).  e.g., --extra-reconall-arg-base=-3T is the correct syntax for adding the stand-alone '-3T' flag to recon-all.  But, --extra-reconall-arg-base='-norm3diters 3' is NOT acceptable.  For recon-all flags that themselves require an argument, you can handle that by specifying  --extra-reconall-arg-base multiple times (in the proper sequential fashion).  e.g., --extra-reconall-arg-base=-norm3diters --extra-reconall-arg-base=3  will be translated to '-norm3diters 3' when passed to recon-all."
-
-opts_AddOptional '--generate_template_only' 'generate_template_only' 'integer' 'Set to 1 to generate template only without timepoints [0]'
-opts_AddOptional '--generate_timepoints_only' 'generate_timepoints_only' 'integer' 'Set to 1 to generate timepoints only (assuming the template has been generated previously with "--generate_template_only" option) [0]'
-
-opts_AddOptional '--extra-reconall-arg-long' 'extra_reconall_arg_long' 'token' "(repeatable)  Generic single token argument to pass to recon-all for the actual longitudinal processing.  See the description for extra_reconall_arg_base parameter for extra details."
-
+opts_AddOptional '--generate-template' 'generate_template' 'integer' 'Set to 0 to not generate template (assuming it is already generated) [1]'
+opts_AddOptional '--generate-timepoints' 'generate_timepoints' 'integer' 'Set to 0 to not generate timepoints [1]'
```

Comment:
These are missing defaults, and don't appear to use `opts_StringToBool`, or any sanity checking.